src/main/java/edu/cornell/kfs/sys/businessobject/actions/CreateDoneBatchFileActionsProvider.java

```diff
@@ -31,17 +32,12 @@
     protected CreateDoneBatchFileAuthorizationService createDoneAuthorizationService;
     
     @Override
-    public List<Map<String, Object>> getActionLinks(BusinessObjectBase businessObject, Person user) {
+    public List<Action> getActionLinks(BusinessObjectBase businessObject, Person user) {
         BatchFile batchFile = (BatchFile) businessObject;
-        List<Map<String, Object>> actionLinks = new LinkedList<>();
+        List<Action> actionLinks = new LinkedList<>();
 
         if (canCreateDoneFile(batchFile, user)) {
-            Map<String, Object> createDoneLink = new LinkedHashMap<>();
-            createDoneLink.put(CUKFSPropertyConstants.LOOKUP_RESULT_ACTION_LABEL, "Create Done");
-            createDoneLink.put(CUKFSPropertyConstants.LOOKUP_RESULT_ACTION_URL,
-                    KRAD_URL_PREFIX + getCreateDoneUrl(batchFile));
-            createDoneLink.put(CUKFSPropertyConstants.LOOKUP_RESULT_ACTION_METHOD, "GET");
-            actionLinks.add(createDoneLink);
+            actionLinks.add(new Action("Create Done", "GET", getCreateDoneUrl(batchFile)));
```

Comment:
This update also seems to remove the "kr/" prefix from the generated URL. Is this intentional? Was the extra prefix the cause of our recent create-done-file issues?

Reply:
Adding that back in. That is not the cause of the recent create done file.